### DISCUSSION TITLE
bpt + CMake + PMM + non-default repo?

### DISCUSSION THREAD

**Discussion by david-bakin** [vector-of-bool/dds#136] [Q&A] [answered]
using `pmm` in `cmake` with `bpt`: how to specify any additional repo(s)?  (would have used the `--use-repo` argument to `bpt build` _outside_ of `cmake` of course)

**Comment by vector-of-bool** [comment, marked as answer]
Work-around answer: you can pass --use-repo as an argument to DEPENDENCIES. In the future, this will warrant a dedicated argument to pmm().
